reproducer.yml

```diff
@@ -55,3 +55,9 @@
   roles:
     - role: ci_setup
     - role: reproducer
+
+  post_tasks:
+    - name: Allow traffic from OSP VMs to OSP API (needed for shiftstack)
+      ansible.builtin.shell: iptables -I LIBVIRT_FWI 1 -o ocpbm -j ACCEPT
+      become: true
+      when: cifmw_allow_vms_to_reach_osp_api
```

Comment:
This is needed for allowing the VMs to contact the OSP API, which is a requirement for the shiftstack OCP nodes. This workload needs to manage OSP resources during both the installation process and the shiftstack operators activity.

Reply:
I'd rather not use `shell` here:
- there's a [module for iptables](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/iptables_module.html) ( ansible-lint will crash for this)
- if really, `ansible.builtin.command` is enough (ansible-lint will crash for this)

Also, the `when` condition will fail if the parameter isn't set, so you want `| default(false) | bool`.
Maybe it should be done in any cases? Not really sure what this would allow, to be fair.

Reply:
iptables module does not look to support the out_interface param for LIBVIRT_FWI (https://docs.ansible.com/ansible/latest/collections/ansible/builtin/iptables_module.html#parameter-out_interface). Moving to command module.

Reply:
dang... do they really filter the chains?